staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1095,9 +1095,12 @@ type ManagedFieldsEntry struct {
 	// Time is timestamp of when these fields were set. It should always be empty if Operation is 'Apply'
 	// +optional
 	Time *Time `json:"time,omitempty" protobuf:"bytes,4,opt,name=time"`
-	// Fields identifies a set of fields.
+
+	// +k8s:deprecated=fields,protobuf=5
+
+	// FieldSet identifies a set of fields.
```

Comment:
We need to document this, either by doing so here or by directing people to smd (and document there).